### ISSUE TITLE
why my loss is not stable

### ISSUE THREAD

**Issue by sunrise666** [balancap/SSD-Tensorflow#272]
INFO:tensorflow:global step 52620: loss = 45.7045 (0.686 sec/step)
INFO:tensorflow:global step 52630: loss = 34.3893 (0.697 sec/step)
INFO:tensorflow:global step 52640: loss = 18.0897 (0.762 sec/step)
INFO:tensorflow:global step 52650: loss = 20.7888 (0.683 sec/step)
INFO:tensorflow:global step 52660: loss = 21.9407 (0.698 sec/step)
INFO:tensorflow:global step 52670: loss = 19.0137 (0.767 sec/step)
INFO:tensorflow:global step 52680: loss = 39.2736 (0.697 sec/step)
INFO:tensorflow:global step 52690: loss = 24.1573 (0.687 sec/step)
INFO:tensorflow:global step 52700: loss = 32.9051 (0.727 sec/step)
INFO:tensorflow:global step 52710: loss = 20.8466 (0.698 sec/step)
INFO:tensorflow:global step 52720: loss = 12.3382 (0.705 sec/step)
INFO:tensorflow:global step 52730: loss = 19.7648 (0.716 sec/step)
INFO:tensorflow:global step 52740: loss = 29.3954 (0.688 sec/step)
INFO:tensorflow:global step 52750: loss = 26.7160 (0.706 sec/step)
INFO:tensorflow:global step 52760: loss = 32.6645 (0.716 sec/step)
INFO:tensorflow:global step 52770: loss = 53.2696 (0.686 sec/step)
INFO:tensorflow:global step 52780: loss = 41.5739 (0.706 sec/step)
INFO:tensorflow:global step 52790: loss = 53.7412 (0.688 sec/step)
INFO:tensorflow:global step 52800: loss = 26.5315 (0.720 sec/step)
INFO:tensorflow:global step 52810: loss = 28.6309 (0.685 sec/step)
INFO:tensorflow:global step 52820: loss = 24.0381 (0.747 sec/step)

**Comment by Just-A-Peek** [comment]
we are the same.

**Comment by woshilicong33** [comment]
we are the same.

**Comment by SWX-TJ** [comment]
Have any solution?

**Comment by sunsmarterjie** [comment]
anyone solve ?

**Comment by navy63** [comment]
The same problem.
